src/features/farming/crops/components/RemoveCrop.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+import { Panel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+import close from "assets/icons/close.png";
+import rustyShovel from "assets/tools/rusty_shovel.png";
+
+interface Props {
+  doShow: boolean;
+  onConfirm: () => void;
+  onClose: () => void;
+}
+
+export const RemoveCrop: React.FC<Props> = ({ doShow, onConfirm, onClose }) => {
+  return (
+    <Modal centered show={doShow} onHide={onClose}>
+      <Panel>
+        <img
+          src={close}
+          className="h-6 top-4 right-4 absolute cursor-pointer"
+          onClick={onClose}
+        />
+        <div className="p-2">
+          <h1 className="text-xl text-center">Remove Growing Crops?</h1>
+          <div className="flex my-4 justify-center">
+            <img src={rustyShovel} style={{ width: "50px" }} />
+          </div>
+          <p className="text-sm mb-4">
+            Warning! You have your shovel equipped.
+          </p>
+          <p className="text-sm mb-3">
+            Are you sure you want to start removing crops? You will not recover
```

Comment:
Also here maybe `lose` instead of `not recover`. 

`You will lose both the crop and its seed!`, Also we can just use string interpolation here instead of the `&apos;` code.

Reply:
In fact, I just discovered the word "it's" is incorrect grammar here anyways. "it's" should only ever represent "it is". I will simply change to "its" instead :+1: .